src/web/app/components/user-notifications-list/user-notifications-list.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="row mb-3" *ngIf="notificationTabs.length > 0">
+<div class="row mb-3" [id]="timezone" *ngIf="notificationTabs.length > 0">
```

Comment:
Any reason why the timezone is used as ID?

Reply:
This is so that the E2E tests can access the timezone that is used to format the start/end date for the notification. I've put it as an ID here so that it won't be shown to the user; should I add a line to display the timezone to the user instead (e.g. `All dates are displayed in {timezone} time.`), similar to the admin notifications page?